### PR TITLE
Stub Routes for Docket Change Functionality

### PR DESCRIPTION
**Pull request by jcq**
Connects #15257

### Description
This adds/updates task actions for docket change tasks, and adds stub routes for them on the frontend.

### Acceptance Criteria
- [ ] Code compiles correctly
- [x] Entries in `TASK_ACTIONS` for the three docket change tasks
- [x] Stub routes created for three docket change tasks
- [x] The one existing task (`DocketSwitchMailTask`) routes correctly (to its stub)

### Notes
This is blocked by https://github.com/department-of-veterans-affairs/caseflow-frontend-toolkit/pull/63

### CODE REVIEW

File: client/app/components/PageRoute.jsx [hunk 1]
```diff
@@ -17,15 +17,6 @@ const PageRoute = (props) => {
     throw new Error('PageRoute must implement `title`');
   }
 
-  if (!props.render && !props.component) {
```
Comment (meckilabritt-navapbc) [inline client/app/components/PageRoute.jsx:20]: Why was this error being thrown before?

File: client/package.json [hunk 2]
```diff
@@ -56,7 +56,7 @@
     "@babel/preset-react": "^7.9.4",
     "@babel/register": "^7.9.0",
     "@babel/runtime-corejs2": "^7.9.2",
-    "@department-of-veterans-affairs/caseflow-frontend-toolkit": "https://github.com/department-of-veterans-affairs/caseflow-frontend-toolkit#81abd0f",
+    "@department-of-veterans-affairs/caseflow-frontend-toolkit": "https://github.com/department-of-veterans-affairs/caseflow-frontend-toolkit#226df5a",
```
Comment (jcq) [inline client/package.json:59]: This includes updated `Breadcrumbs` component; necessary for allowing the use of the now-standard syntax of using `children` prop for `react-router` routes

File: spec/models/tasks/docket_switch_mail_task_spec.rb [hunk 3]
```diff
@@ -57,7 +57,7 @@
         after { FeatureToggle.disable!(:docket_change) }
 
         it "returns the available_actions as defined by Task" do
-          expect(subject).to eq(task_actions + [Constants.TASK_ACTIONS.SWITCH_DOCKET.to_h])
+          expect(subject).to eq(task_actions + [Constants.TASK_ACTIONS.DOCKET_CHANGE_SEND_TO_JUDGE.to_h])
```
Comment (jcq) [inline spec/models/tasks/docket_switch_mail_task_spec.rb:60]: This just clarified the purpose of the task, after our requirements changed and more tasks were added

File: client/constants/TASK_ACTIONS.json [hunk 4]
```diff
@@ -80,9 +80,17 @@
     "value": "sign_motion_to_vacate",
     "func": "sign_motion_to_vacate_data"
   },
-  "SWITCH_DOCKET": {
-    "label": "Switch Docket",
-    "value": "switch_docket"
+  "DOCKET_CHANGE_SEND_TO_JUDGE": {
```
Comment (jcq) [inline client/constants/TASK_ACTIONS.json:83]: Here we update to include the three tasks that are now known to be needed for docket change functionality

File: client/app/components/PageRoute.jsx (hunk 1 above)
Comment (jcq) [reply client/app/components/PageRoute.jsx:20]: Previously the guidance from `react-router` was to use either the `render` or `component` props. That has changed with the new version of `react-router`, and they now advise rendering the contents of a given route within `children`. This is currently optional, but in 6.x+ the others become deprecated (and perhaps removed?).

File: client/app/queue/docketChange/docketChangeRoutes.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+import TASK_ACTIONS from '../../../constants/TASK_ACTIONS';
+
+const PageRoutes = [
+  <PageRoute
+    path={`/queue/appeals/:appealId/tasks/:taskId/${
+      TASK_ACTIONS.DOCKET_CHANGE_SEND_TO_JUDGE.value
+    }`}
+    title={`${TASK_ACTIONS.DOCKET_CHANGE_SEND_TO_JUDGE.label} | Caseflow`}
+  >
+    {/* Replace with actual component */}
```
Comment (meckilabritt-navapbc) [inline client/app/queue/docketChange/docketChangeRoutes.js:14]: What is this?

Comment (jcq) [reply client/app/queue/docketChange/docketChangeRoutes.js:14]: This is just the stub, providing a place for future PRs to add actual components

File: client/app/components/PageRoute.jsx (hunk 1 above)
Comment (meckilabritt-navapbc) [reply client/app/components/PageRoute.jsx:20]: Thanks for the clarification

File: client/app/queue/docketChange/docketChangeRoutes.js (hunk 5 above)
Comment (meckilabritt-navapbc) [reply client/app/queue/docketChange/docketChangeRoutes.js:14]: Ahh, understood

**Review by meckilabritt-navapbc — APPROVED** [review]

File: client/constants/TASK_ACTIONS.json [hunk 6]
```diff
@@ -80,9 +80,17 @@
...
+    "label": "Send to Judge",
+    "value": "docket_change/recommend"
```
Comment (leikkisa) [inline client/constants/TASK_ACTIONS.json:85]: If we're calling it "docket switch" from the user's point of view, does it make sense for the routes to match?

**Review by leikkisa — APPROVED** [review]
Hey JC - This looks great! I added some comments, but nothing blocking.

Lmk if you do want to smoke test in UAT though.

**Comment by leikkisa** [comment]
Also, tested locally. Fun!

File: client/constants/TASK_ACTIONS.json [hunk 7]
```diff
@@ -80,9 +80,17 @@
...
+  },
+  "DOCKET_CHANGE_JUDGE_RULING": {
+    "label": "Send to Judge",
```
[the remark this answers is not in the document]
Comment (jcq) [reply client/constants/TASK_ACTIONS.json:88]: This is the current copy from Rutvi, but we'll update if need be after further user research.

File: spec/models/tasks/docket_switch_mail_task_spec.rb (hunk 3 above)
[the remark this answers is not in the document]
Comment (jcq) [reply spec/models/tasks/docket_switch_mail_task_spec.rb:60]: Agreed. :\

File: client/package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (jcq) [reply client/package.json:59]: I think the more recent commit was actually just a dependabot upgrade to one of the deps. Doesn't have an actual effect on the code

**Pull request merged**